bitnami/tomcat/templates/NOTES.txt

```diff
@@ -3,7 +3,14 @@
 
 1. Get the Tomcat URL by running:
 
-{{- if contains "NodePort" .Values.service.type }}
+{{- if .Values.ingress.enabled }}
+
+export CLUSTER_IP=$(minikube ip) # On Minikube. Use: `kubectl cluster-info` on others K8s clusters
```

Comment:
This documentation should not be specific to minikube.

Reply:
If ingress is enabled I would just print the value of the host configured in the values.yaml